components/git/land.js

```diff
@@ -38,6 +38,8 @@ function builder(yargs) {
     .options(landOptions).positional('prid', {
       describe: 'ID or URL of the Pull Request'
     })
+    .boolean('non-interactive')
```

Comment:
I'd actually prefer `--yes`/`-y` here...I think that's a more universal option name?

Reply:
Humm, that makes sense. I got `non-interactive` from debconf's non-interactive mode, but `--yes/-y` seems to be used by more tools. 

But assuming `yes` to all questions is not the behavior I was looking for. The idea is to run all actions from `git node land` without asking on the best case scenario (all validations passed, only one commit, etc.). So, for example, if CI failed, it shouldn't continue.

We could still use `--yes` for that though.

Reply:
Can we add some kind of config file for the behavior of `--yes`? That would also make it easier to tweak stuff in bots I suppose.

Reply:
Maybe `--silent` is also an appropriate name, and the config argument could be specified by something like `--silent-answers`.